packages/core/watcher/src/Watcher.js

```diff
@@ -149,10 +167,11 @@ class Watcher extends EventEmitter {
     });
   }
 
-  getWatched() {
+  getWatched(): {[string]: []} {
```

Comment:
This api is kind of weird. It returns an object mapping paths to empty arrays. Are these intended to be mutated later or something? @DeMoorJasper?